scripts/create_sample_meta.py

```diff
@@ -33,10 +33,28 @@ def save_dataframe(df, build_dir):
 
 
 def rename_columns(df):
-    rename_map = {"index1": "index_1", "index2": "index_2", "pert_id": "compound_id"}
+    rename_map = {"index1": "index_1", "index2": "index_2", "trt_type": "pert_type",
+                  "treatment": "pert_name", "dose": "pert_dose", "dose_unit": "pert_dose_unit"}
     return df.rename(columns=rename_map)
 
 
+def add_control_barcodes(df, control_barcodes):
+    control_barcode_df = df.copy()
+    control_barcode_df["cb_ladder"] = control_barcodes
+    return control_barcode_df
+
+
+def replace_pert_type(df):
+    df["pert_type"] = df["pert_type"].replace(
```

Comment:
Is this renaming actually required? Jenkins accepts ctl_vehicle as the negcon parameter for sushi to handle this flexibly.

Asking because if we rename it here, then we have to reverse the renaming in the adapter script since the analysis pipeline still uses ctl_vehicle. and that will become confusing

Reply:
Yes, I think we mentioned this yesterday. I'm going to remove the pert_type renaming entirely and have the negcon param default to ctl_vehicle (output by COMET)